scripts/example/jupyterlab_version/l8_cloud_train_network.yaml

```diff
@@ -0,0 +1,115 @@
+train:
+  loss: sparse_categorical_crossentropy 
+  metrics:
+    - sparse_categorical_accuracy
+  network:
+    layers:
```

Comment:
Could you separate out the architecture into a separate file? I think it would look a lot less intimidating.